Meteo France flaky test

<!-- READ THIS FIRST:
  - If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
  - Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
  - Do not report issues for integrations if you are using custom components or integrations.
  - Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
  DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
-->
## The problem
<!-- 
  Describe the issue you are experiencing here to communicate to the
  maintainers. Tell us what you were trying to do and what happened instead.
-->
The meteo_france integration produces a flaky test.

## Environment
<!--
  Provide details about the versions you are using, which helps us to reproduce
  and find the issue quicker. Version information is found in the
  Home Assistant frontend: Developer tools -> Info.
-->

- Home Assistant release with the issue: 0.106.0.dev0
- Last working Home Assistant release (if known): N/A
- Operating environment (Hass.io/Docker/Windows/etc.): Azure CI
- Integration causing this issue: Meteo France
- Link to integration documentation on our website: https://www.home-assistant.io/integrations/meteo_france/

## Problem-relevant `configuration.yaml`
<!--
  An example configuration that caused the problem for you. Fill this out even
  if it seems unimportant to you. Please be sure to remove personal information
  like passwords, private URLs and other credentials.
-->

```yaml

```

## Traceback/Error logs
<!--
  If you come across any trace or error logs, please provide them.
-->

```txt
2020-02-09T18:17:58.2142746Z =================================== FAILURES ===================================
2020-02-09T18:17:58.2142948Z _____________________________ test_import[pyloop] ______________________________
2020-02-09T18:17:58.2143680Z [gw1] linux -- Python 3.7.0 /__w/1/s/venv/bin/python
2020-02-09T18:17:58.2143860Z 
2020-02-09T18:17:58.2143977Z pyfuncitem = <Function test_import[pyloop]>
2020-02-09T18:17:58.2144543Z 
2020-02-09T18:17:58.2145080Z     def pytest_pyfunc_call(pyfuncitem):  # type: ignore
2020-02-09T18:17:58.2145298Z         """
2020-02-09T18:17:58.2145503Z         Run coroutines in an event loop instead of a normal function call.
2020-02-09T18:17:58.2145714Z         """
2020-02-09T18:17:58.2146251Z         fast = pyfuncitem.config.getoption("--aiohttp-fast")
2020-02-09T18:17:58.2146528Z         if asyncio.iscoroutinefunction(pyfuncitem.function):
2020-02-09T18:17:58.2146964Z             existing_loop = pyfuncitem.funcargs.get('proactor_loop')\
2020-02-09T18:17:58.2147420Z                 or pyfuncitem.funcargs.get('loop', None)
2020-02-09T18:17:58.2147646Z             with _runtime_warning_context():
2020-02-09T18:17:58.2147866Z                 with _passthrough_loop_context(existing_loop, fast=fast) as _loop:
2020-02-09T18:17:58.2148091Z                     testargs = {arg: pyfuncitem.funcargs[arg]
2020-02-09T18:17:58.2148297Z                                 for arg in pyfuncitem._fixtureinfo.argnames}
2020-02-09T18:17:58.2148524Z >                   _loop.run_until_complete(pyfuncitem.obj(**testargs))
2020-02-09T18:17:58.2148705Z 
2020-02-09T18:17:58.2149124Z venv/lib/python3.7/site-packages/aiohttp/pytest_plugin.py:170: 
2020-02-09T18:17:58.2149368Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2020-02-09T18:17:58.2149589Z /usr/local/lib/python3.7/asyncio/base_events.py:555: in run_until_complete
2020-02-09T18:17:58.2149793Z     self.run_forever()
2020-02-09T18:17:58.2150004Z /usr/local/lib/python3.7/asyncio/base_events.py:523: in run_forever
2020-02-09T18:17:58.2150199Z     self._run_once()
2020-02-09T18:17:58.2150402Z /usr/local/lib/python3.7/asyncio/base_events.py:1722: in _run_once
2020-02-09T18:17:58.2150601Z     event_list = self._selector.select(timeout)
2020-02-09T18:17:58.2150811Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2020-02-09T18:17:58.2151007Z 
2020-02-09T18:17:58.2151445Z self = <selectors.EpollSelector object at 0x7f0ea57910b8>, timeout = -1
2020-02-09T18:17:58.2151652Z 
2020-02-09T18:17:58.2151846Z     def select(self, timeout=None):
2020-02-09T18:17:58.2152043Z         if timeout is None:
2020-02-09T18:17:58.2152424Z             timeout = -1
2020-02-09T18:17:58.2152645Z         elif timeout <= 0:
2020-02-09T18:17:58.2152842Z             timeout = 0
2020-02-09T18:17:58.2153307Z         else:
2020-02-09T18:17:58.2153511Z             # epoll_wait() has a resolution of 1 millisecond, round away
2020-02-09T18:17:58.2153718Z             # from zero to wait *at least* timeout seconds.
2020-02-09T18:17:58.2154157Z             timeout = math.ceil(timeout * 1e3) * 1e-3
2020-02-09T18:17:58.2154384Z     
2020-02-09T18:17:58.2154583Z         # epoll_wait() expects `maxevents` to be greater than zero;
2020-02-09T18:17:58.2154870Z         # we want to make sure that `select()` can be called when no
2020-02-09T18:17:58.2155031Z         # FD is registered.
2020-02-09T18:17:58.2155189Z         max_ev = max(len(self._fd_to_key), 1)
2020-02-09T18:17:58.2155450Z     
2020-02-09T18:17:58.2155591Z         ready = []
2020-02-09T18:17:58.2155746Z         try:
2020-02-09T18:17:58.2155900Z >           fd_event_list = self._selector.poll(timeout, max_ev)
2020-02-09T18:17:58.2156065Z E           Failed: Timeout >9.0s
2020-02-09T18:17:58.2156196Z 
2020-02-09T18:17:58.2156355Z /usr/local/lib/python3.7/selectors.py:468: Failed
2020-02-09T18:17:58.2156721Z ---------------------------- Captured stderr setup -----------------------------
2020-02-09T18:17:58.2156936Z DEBUG:asyncio:Using selector: EpollSelector
2020-02-09T18:17:58.2157299Z ------------------------------ Captured log setup ------------------------------
2020-02-09T18:17:58.2157493Z DEBUG    asyncio:selector_events.py:64 Using selector: EpollSelector
2020-02-09T18:17:58.2157846Z ----------------------------- Captured stderr call -----------------------------
2020-02-09T18:17:58.2158045Z INFO:homeassistant.loader:Loaded test from custom_components.test
2020-02-09T18:17:58.2158333Z INFO:homeassistant.loader:Loaded test_package from custom_components.test_package
2020-02-09T18:17:58.2158491Z INFO:homeassistant.loader:Loaded meteo_france from homeassistant.components.meteo_france
2020-02-09T18:17:58.2158640Z INFO:homeassistant.setup:Setting up meteo_france
2020-02-09T18:17:58.2158783Z INFO:homeassistant.setup:Setup of domain meteo_france took 0.0 seconds.
2020-02-09T18:17:58.2158914Z 
2020-02-09T18:17:58.2159050Z +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
2020-02-09T18:17:58.2159177Z 
2020-02-09T18:17:58.2159311Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_8 (139700035639040) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2159462Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2159600Z     self._bootstrap_inner()
2020-02-09T18:17:58.2159741Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2159883Z     self.run()
2020-02-09T18:17:58.2160015Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2160165Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2160311Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2160452Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2160570Z 
2020-02-09T18:17:58.2160713Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_7 (139700044031744) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2160878Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2161014Z     self._bootstrap_inner()
2020-02-09T18:17:58.2161153Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2161296Z     self.run()
2020-02-09T18:17:58.2161429Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2161571Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2161710Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2161861Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2161987Z 
2020-02-09T18:17:58.2162122Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_6 (139700052424448) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2162271Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2162407Z     self._bootstrap_inner()
2020-02-09T18:17:58.2162631Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2162910Z     self.run()
2020-02-09T18:17:58.2163086Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2163230Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2163371Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2163518Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2163637Z 
2020-02-09T18:17:58.2163771Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_5 (139700505138944) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2163994Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2164130Z     self._bootstrap_inner()
2020-02-09T18:17:58.2164275Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2164419Z     self.run()
2020-02-09T18:17:58.2164567Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2164721Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2164860Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2165007Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2165118Z 
2020-02-09T18:17:58.2165260Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_4 (139700898215680) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2165412Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2165548Z     self._bootstrap_inner()
2020-02-09T18:17:58.2165689Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2165837Z     self.run()
2020-02-09T18:17:58.2165966Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2166109Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2166248Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 80, in _worker
2020-02-09T18:17:58.2166399Z     work_item.run()
2020-02-09T18:17:58.2166540Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
2020-02-09T18:17:58.2166681Z     result = self.fn(*self.args, **self.kwargs)
2020-02-09T18:17:58.2167070Z   File "/__w/1/s/venv/lib/python3.7/site-packages/vigilancemeteo/vigilance_proxy.py", line 130, in update_data
2020-02-09T18:17:58.2167259Z     current_checksum = self._get_new_checksum()
2020-02-09T18:17:58.2167593Z   File "/__w/1/s/venv/lib/python3.7/site-packages/vigilancemeteo/vigilance_proxy.py", line 91, in _get_new_checksum
2020-02-09T18:17:58.2167936Z     text = urlopen(self.URL_VIGILANCE_METEO_CHECKSUM).read().decode("utf-8")
2020-02-09T18:17:58.2168114Z   File "/usr/local/lib/python3.7/urllib/request.py", line 222, in urlopen
2020-02-09T18:17:58.2168253Z     return opener.open(url, data, timeout)
2020-02-09T18:17:58.2168415Z   File "/usr/local/lib/python3.7/urllib/request.py", line 525, in open
2020-02-09T18:17:58.2168554Z     response = self._open(req, data)
2020-02-09T18:17:58.2168703Z   File "/usr/local/lib/python3.7/urllib/request.py", line 543, in _open
2020-02-09T18:17:58.2168968Z     '_open', req)
2020-02-09T18:17:58.2169120Z   File "/usr/local/lib/python3.7/urllib/request.py", line 503, in _call_chain
2020-02-09T18:17:58.2169266Z     result = func(*args)
2020-02-09T18:17:58.2169402Z   File "/usr/local/lib/python3.7/urllib/request.py", line 1345, in http_open
2020-02-09T18:17:58.2169555Z     return self.do_open(http.client.HTTPConnection, req)
2020-02-09T18:17:58.2169705Z   File "/usr/local/lib/python3.7/urllib/request.py", line 1317, in do_open
2020-02-09T18:17:58.2169998Z     encode_chunked=req.has_header('Transfer-encoding'))
2020-02-09T18:17:58.2170164Z   File "/usr/local/lib/python3.7/http/client.py", line 1229, in request
2020-02-09T18:17:58.2170313Z     self._send_request(method, url, body, headers, encode_chunked)
2020-02-09T18:17:58.2170459Z   File "/usr/local/lib/python3.7/http/client.py", line 1275, in _send_request
2020-02-09T18:17:58.2170682Z     self.endheaders(body, encode_chunked=encode_chunked)
2020-02-09T18:17:58.2170831Z   File "/usr/local/lib/python3.7/http/client.py", line 1224, in endheaders
2020-02-09T18:17:58.2170974Z     self._send_output(message_body, encode_chunked=encode_chunked)
2020-02-09T18:17:58.2171125Z   File "/usr/local/lib/python3.7/http/client.py", line 1016, in _send_output
2020-02-09T18:17:58.2171263Z     self.send(msg)
2020-02-09T18:17:58.2171404Z   File "/usr/local/lib/python3.7/http/client.py", line 956, in send
2020-02-09T18:17:58.2171550Z     self.connect()
2020-02-09T18:17:58.2171683Z   File "/usr/local/lib/python3.7/http/client.py", line 928, in connect
2020-02-09T18:17:58.2171882Z     (self.host,self.port), self.timeout, self.source_address)
2020-02-09T18:17:58.2172030Z   File "/usr/local/lib/python3.7/socket.py", line 716, in create_connection
2020-02-09T18:17:58.2172185Z     sock.connect(sa)
2020-02-09T18:17:58.2172304Z 
2020-02-09T18:17:58.2172441Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_3 (139700513793792) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2172599Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2172734Z     self._bootstrap_inner()
2020-02-09T18:17:58.2172875Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2173020Z     self.run()
2020-02-09T18:17:58.2173152Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2173296Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2173437Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2173592Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2173712Z 
2020-02-09T18:17:58.2173847Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_2 (139700880901888) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2173999Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2174134Z     self._bootstrap_inner()
2020-02-09T18:17:58.2174286Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2174429Z     self.run()
2020-02-09T18:17:58.2174560Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2174704Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2174844Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2174993Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2175105Z 
2020-02-09T18:17:58.2175249Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_1 (139700018853632) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2175405Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2175554Z     self._bootstrap_inner()
2020-02-09T18:17:58.2175697Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2175840Z     self.run()
2020-02-09T18:17:58.2175971Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2176120Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2176260Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2176406Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2176518Z 
2020-02-09T18:17:58.2176658Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_0 (139700522186496) ~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2176812Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-02-09T18:17:58.2176946Z     self._bootstrap_inner()
2020-02-09T18:17:58.2177087Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-02-09T18:17:58.2177234Z     self.run()
2020-02-09T18:17:58.2177366Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-02-09T18:17:58.2177511Z     self._target(*self._args, **self._kwargs)
2020-02-09T18:17:58.2177650Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-02-09T18:17:58.2177839Z     work_item = work_queue.get(block=True)
2020-02-09T18:17:58.2177951Z 
2020-02-09T18:17:58.2178093Z ~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139702208648960) ~~~~~~~~~~~~~~~~~~~~~
2020-02-09T18:17:58.2178461Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 285, in _perform_spawn
2020-02-09T18:17:58.2178628Z     reply.run()
2020-02-09T18:17:58.2178943Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 220, in run
2020-02-09T18:17:58.2179115Z     self._result = func(*args, **kwargs)
2020-02-09T18:17:58.2179454Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 967, in _thread_receiver
2020-02-09T18:17:58.2179677Z     msg = Message.from_io(io)
2020-02-09T18:17:58.2179995Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 432, in from_io
2020-02-09T18:17:58.2180169Z     header = io.read(9)  # type 1, channel 4, payload 4
2020-02-09T18:17:58.2180491Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 400, in read
2020-02-09T18:17:58.2180797Z     data = self._read(numbytes - len(buf))
2020-02-09T18:17:58.2180934Z 
2020-02-09T18:17:58.2181069Z +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
2020-02-09T18:17:58.2181384Z ------------------------------ Captured log call -------------------------------
2020-02-09T18:17:58.2181561Z INFO     homeassistant.loader:loader.py:205 Loaded test from custom_components.test
2020-02-09T18:17:58.2181711Z INFO     homeassistant.loader:loader.py:205 Loaded test_package from custom_components.test_package
2020-02-09T18:17:58.2181874Z INFO     homeassistant.loader:loader.py:205 Loaded meteo_france from homeassistant.components.meteo_france
2020-02-09T18:17:58.2182039Z INFO     homeassistant.setup:setup.py:153 Setting up meteo_france
2020-02-09T18:17:58.2182187Z INFO     homeassistant.setup:setup.py:187 Setup of domain meteo_france took 0.0 seconds.
2020-02-09T18:17:58.2182500Z --------------------------- Captured stderr teardown ---------------------------
2020-02-09T18:17:58.2182670Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_stop[L]>
2020-02-09T18:17:58.2182821Z ERROR:homeassistant.core:Error doing job: Task exception was never retrieved
2020-02-09T18:17:58.2182965Z Traceback (most recent call last):
2020-02-09T18:17:58.2183103Z   File "/__w/1/s/homeassistant/core.py", line 1201, in async_call
2020-02-09T18:17:58.2183247Z     raise ServiceNotFound(domain, service) from None
2020-02-09T18:17:58.2183399Z homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/dismiss
2020-02-09T18:17:58.2183572Z DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): www.meteofrance.com:80
2020-02-09T18:17:58.2183942Z DEBUG:urllib3.connectionpool:http://www.meteofrance.com:80 "GET /mf3-rpc-portlet/rest/lieu/facet/previsions/search/74220 HTTP/1.1" 200 None
2020-02-09T18:17:58.2184138Z DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): www.meteofrance.com:80
2020-02-09T18:17:58.2184496Z DEBUG:urllib3.connectionpool:http://www.meteofrance.com:80 "GET /previsions-meteo-france/la-clusaz/74220 HTTP/1.1" 200 None
2020-02-09T18:17:58.2184686Z DEBUG:homeassistant.components.meteo_france:meteo_france sensor platform loaded for 74220
2020-02-09T18:17:58.2184846Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=meteo_france>
2020-02-09T18:17:58.2184994Z DEBUG:homeassistant.core:Bus:Handling <Event config_entry_discovered[L]>
2020-02-09T18:17:58.2185148Z ERROR:homeassistant.core:Error doing job: Task exception was never retrieved
2020-02-09T18:17:58.2185296Z Traceback (most recent call last):
2020-02-09T18:17:58.2185437Z   File "/__w/1/s/homeassistant/core.py", line 1201, in async_call
2020-02-09T18:17:58.2185581Z     raise ServiceNotFound(domain, service) from None
2020-02-09T18:17:58.2185729Z homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/create
2020-02-09T18:17:58.2185886Z INFO:homeassistant.loader:Loaded weather from homeassistant.components.weather
2020-02-09T18:17:58.2186098Z INFO:homeassistant.loader:Loaded sensor from homeassistant.components.sensor
2020-02-09T18:17:58.2186238Z INFO:homeassistant.setup:Setting up weather
2020-02-09T18:17:58.2186382Z INFO:homeassistant.setup:Setup of domain weather took 0.0 seconds.
2020-02-09T18:17:58.2186534Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=weather>
2020-02-09T18:17:58.2186676Z INFO:homeassistant.setup:Setting up sensor
2020-02-09T18:17:58.2186820Z INFO:homeassistant.setup:Setup of domain sensor took 0.0 seconds.
2020-02-09T18:17:58.2186964Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=sensor>
2020-02-09T18:17:58.2187187Z INFO:homeassistant.components.weather:Setting up weather.meteo_france
2020-02-09T18:17:58.2187337Z INFO:homeassistant.components.sensor:Setting up sensor.meteo_france
2020-02-09T18:17:58.2187486Z INFO:homeassistant.components.meteo_france.sensor:Weather alert watcher added for 74220 in department 74
2020-02-09T18:17:58.2187654Z INFO:homeassistant.helpers.entity_registry:Registered new weather.meteo_france entity: weather.la_clusaz
2020-02-09T18:17:58.2187822Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=weather.la_clusaz>
2020-02-09T18:17:58.2188190Z DEBUG:homeassistant.components.meteo_france.weather:reftime used for La-Clusaz forecast: 2020-02-10 12:00:00+00:00
2020-02-09T18:17:58.2188566Z DEBUG:homeassistant.components.meteo_france.weather:reftime used for La-Clusaz forecast: 2020-02-10 12:00:00+00:00
2020-02-09T18:17:58.2189522Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=weather.la_clusaz, old_state=None, new_state=<state weather.la_clusaz=cloudy; temperature=4, wind_bearing=S, wind_speed=10, attribution=Data provided by MÃ©tÃ©o-France, forecast=[{'datetime': '2020-02-10T12:00:00+00:00', 'temperature': 9, 'templow': 3, 'condition': 'rainy'}, {'datetime': '2020-02-11T12:00:00+00:00', 'temperature': 5, 'templow': 1, 'condition': 'rainy'}, {'datetime': '2020-02-12T12:00:00+00:00', 'temperature': 5, 'templow': -1, 'condition': 'snowy'}, {'datetime': '2020-02-13T12:00:00+00:00', 'temperature': 6, 'templow': -2, 'condition': 'sunny'}, {'datetime': '2020-02-14T12:00:00+00:00', 'temperature': 4, 'templow': -1, 'condition': 'cloudy'}], friendly_name=La-Clusaz @ 2020-02-09T18:05:40.356499+00:00>>
2020-02-09T18:17:58.2189928Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_weather
2020-02-09T18:17:58.2190128Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_weather>
2020-02-09T18:17:58.2190663Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_weather, old_state=None, new_state=<state sensor.la_clusaz_weather=TrÃ¨s nuageux; attribution=Data provided by MÃ©tÃ©o-France, friendly_name=La-Clusaz Weather, icon=mdi:weather-partly-cloudy @ 2020-02-09T18:05:40.357537+00:00>>
2020-02-09T18:17:58.2190925Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_rain_chance
2020-02-09T18:17:58.2191109Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_rain_chance>
2020-02-09T18:17:58.2191649Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_rain_chance, old_state=None, new_state=<state sensor.la_clusaz_rain_chance=20; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Rain chance, icon=mdi:weather-rainy @ 2020-02-09T18:05:40.358920+00:00>>
2020-02-09T18:17:58.2191909Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_uv
2020-02-09T18:17:58.2192084Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_uv>
2020-02-09T18:17:58.2192579Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_uv, old_state=None, new_state=<state sensor.la_clusaz_uv=2; attribution=Data provided by MÃ©tÃ©o-France, friendly_name=La-Clusaz UV, icon=mdi:sunglasses @ 2020-02-09T18:05:40.359973+00:00>>
2020-02-09T18:17:58.2192902Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_freeze_chance
2020-02-09T18:17:58.2193081Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_freeze_chance>
2020-02-09T18:17:58.2193622Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_freeze_chance, old_state=None, new_state=<state sensor.la_clusaz_freeze_chance=0; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Freeze chance, icon=mdi:snowflake @ 2020-02-09T18:05:40.361123+00:00>>
2020-02-09T18:17:58.2193931Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_wind_speed
2020-02-09T18:17:58.2194116Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_wind_speed>
2020-02-09T18:17:58.2194650Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_wind_speed, old_state=None, new_state=<state sensor.la_clusaz_wind_speed=10; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=km/h, friendly_name=La-Clusaz Wind Speed, icon=mdi:weather-windy @ 2020-02-09T18:05:40.362469+00:00>>
2020-02-09T18:17:58.2194908Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_thunder_chance
2020-02-09T18:17:58.2195095Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_thunder_chance>
2020-02-09T18:17:58.2195657Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_thunder_chance, old_state=None, new_state=<state sensor.la_clusaz_thunder_chance=0; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Thunder chance, icon=mdi:weather-lightning @ 2020-02-09T18:05:40.363769+00:00>>
2020-02-09T18:17:58.2196085Z ERROR:homeassistant.components.meteo_france.sensor:No condition next_rain for location La-Clusaz
2020-02-09T18:17:58.2196279Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_next_rain
2020-02-09T18:17:58.2196454Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_next_rain>
2020-02-09T18:17:58.2196980Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_next_rain, old_state=None, new_state=<state sensor.la_clusaz_next_rain=unknown; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=min, friendly_name=La-Clusaz Next rain, icon=mdi:weather-rainy @ 2020-02-09T18:05:40.365052+00:00>>
2020-02-09T18:17:58.2197442Z DEBUG:homeassistant.components.meteo_france.sensor:weather alert watcher for La-Clusaz updated. Proxy have the status: checksum_cached_60s
2020-02-09T18:17:58.2197649Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_weather_alert
2020-02-09T18:17:58.2197825Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_weather_alert>
2020-02-09T18:17:58.2198519Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_weather_alert, old_state=None, new_state=<state sensor.la_clusaz_weather_alert=Jaune; Bulletin date=2020-02-09T15:05:00+00:00, Vent violent=Jaune, Pluie-innodation=Vert, Orages=Vert, Inondation=Vert, Neige-verglas=Vert, Canicule=Vert, Grand-froid=Vert, Avalanches=Jaune, Vagues-submersion=Vert, attribution=Data provided by MÃ©tÃ©o-France, friendly_name=La-Clusaz Weather Alert, icon=mdi:weather-cloudy-alert @ 2020-02-09T18:05:40.366510+00:00>>
2020-02-09T18:17:58.2198922Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_snow_chance
2020-02-09T18:17:58.2199102Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_snow_chance>
2020-02-09T18:17:58.2199638Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_snow_chance, old_state=None, new_state=<state sensor.la_clusaz_snow_chance=10; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Snow chance, icon=mdi:weather-snowy @ 2020-02-09T18:05:40.367746+00:00>>
2020-02-09T18:17:58.2199945Z INFO:homeassistant.helpers.entity_registry:Registered new sensor.meteo_france entity: sensor.la_clusaz_temperature
2020-02-09T18:17:58.2200129Z DEBUG:homeassistant.core:Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_temperature>
2020-02-09T18:17:58.2200690Z DEBUG:homeassistant.core:Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_temperature, old_state=None, new_state=<state sensor.la_clusaz_temperature=4; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=Â°C, friendly_name=La-Clusaz Temperature, icon=mdi:thermometer, device_class=temperature @ 2020-02-09T18:05:40.368917+00:00>>
2020-02-09T18:17:58.2200969Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_close[L]>
2020-02-09T18:17:58.2201296Z ---------------------------- Captured log teardown -----------------------------
2020-02-09T18:17:58.2201477Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event homeassistant_stop[L]>
2020-02-09T18:17:58.2201627Z ERROR    homeassistant.core:core.py:139 Error doing job: Task exception was never retrieved
2020-02-09T18:17:58.2201784Z Traceback (most recent call last):
2020-02-09T18:17:58.2201922Z   File "/__w/1/s/homeassistant/core.py", line 1201, in async_call
2020-02-09T18:17:58.2202070Z     raise ServiceNotFound(domain, service) from None
2020-02-09T18:17:58.2202223Z homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/dismiss
2020-02-09T18:17:58.2202386Z DEBUG    urllib3.connectionpool:connectionpool.py:225 Starting new HTTP connection (1): www.meteofrance.com:80
2020-02-09T18:17:58.2202955Z DEBUG    urllib3.connectionpool:connectionpool.py:437 http://www.meteofrance.com:80 "GET /mf3-rpc-portlet/rest/lieu/facet/previsions/search/74220 HTTP/1.1" 200 None
2020-02-09T18:17:58.2203201Z DEBUG    urllib3.connectionpool:connectionpool.py:225 Starting new HTTP connection (1): www.meteofrance.com:80
2020-02-09T18:17:58.2203594Z DEBUG    urllib3.connectionpool:connectionpool.py:437 http://www.meteofrance.com:80 "GET /previsions-meteo-france/la-clusaz/74220 HTTP/1.1" 200 None
2020-02-09T18:17:58.2203790Z DEBUG    homeassistant.components.meteo_france:__init__.py:76 meteo_france sensor platform loaded for 74220
2020-02-09T18:17:58.2203954Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event component_loaded[L]: component=meteo_france>
2020-02-09T18:17:58.2204123Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event config_entry_discovered[L]>
2020-02-09T18:17:58.2204275Z ERROR    homeassistant.core:core.py:139 Error doing job: Task exception was never retrieved
2020-02-09T18:17:58.2204605Z Traceback (most recent call last):
2020-02-09T18:17:58.2204795Z   File "/__w/1/s/homeassistant/core.py", line 1201, in async_call
2020-02-09T18:17:58.2204970Z     raise ServiceNotFound(domain, service) from None
2020-02-09T18:17:58.2205145Z homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/create
2020-02-09T18:17:58.2205331Z INFO     homeassistant.loader:loader.py:205 Loaded weather from homeassistant.components.weather
2020-02-09T18:17:58.2205518Z INFO     homeassistant.loader:loader.py:205 Loaded sensor from homeassistant.components.sensor
2020-02-09T18:17:58.2205696Z INFO     homeassistant.setup:setup.py:153 Setting up weather
2020-02-09T18:17:58.2205865Z INFO     homeassistant.setup:setup.py:187 Setup of domain weather took 0.0 seconds.
2020-02-09T18:17:58.2206155Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event component_loaded[L]: component=weather>
2020-02-09T18:17:58.2206346Z INFO     homeassistant.setup:setup.py:153 Setting up sensor
2020-02-09T18:17:58.2206515Z INFO     homeassistant.setup:setup.py:187 Setup of domain sensor took 0.0 seconds.
2020-02-09T18:17:58.2206703Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event component_loaded[L]: component=sensor>
2020-02-09T18:17:58.2206890Z INFO     homeassistant.components.weather:entity_platform.py:166 Setting up weather.meteo_france
2020-02-09T18:17:58.2207068Z INFO     homeassistant.components.sensor:entity_platform.py:166 Setting up sensor.meteo_france
2020-02-09T18:17:58.2207387Z INFO     homeassistant.components.meteo_france.sensor:sensor.py:49 Weather alert watcher added for 74220 in department 74
2020-02-09T18:17:58.2207595Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new weather.meteo_france entity: weather.la_clusaz
2020-02-09T18:17:58.2207802Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=weather.la_clusaz>
2020-02-09T18:17:58.2208250Z DEBUG    homeassistant.components.meteo_france.weather:weather.py:97 reftime used for La-Clusaz forecast: 2020-02-10 12:00:00+00:00
2020-02-09T18:17:58.2208705Z DEBUG    homeassistant.components.meteo_france.weather:weather.py:97 reftime used for La-Clusaz forecast: 2020-02-10 12:00:00+00:00
2020-02-09T18:17:58.2209827Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=weather.la_clusaz, old_state=None, new_state=<state weather.la_clusaz=cloudy; temperature=4, wind_bearing=S, wind_speed=10, attribution=Data provided by MÃ©tÃ©o-France, forecast=[{'datetime': '2020-02-10T12:00:00+00:00', 'temperature': 9, 'templow': 3, 'condition': 'rainy'}, {'datetime': '2020-02-11T12:00:00+00:00', 'temperature': 5, 'templow': 1, 'condition': 'rainy'}, {'datetime': '2020-02-12T12:00:00+00:00', 'temperature': 5, 'templow': -1, 'condition': 'snowy'}, {'datetime': '2020-02-13T12:00:00+00:00', 'temperature': 6, 'templow': -2, 'condition': 'sunny'}, {'datetime': '2020-02-14T12:00:00+00:00', 'temperature': 4, 'templow': -1, 'condition': 'cloudy'}], friendly_name=La-Clusaz @ 2020-02-09T18:05:40.356499+00:00>>
2020-02-09T18:17:58.2210320Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_weather
2020-02-09T18:17:58.2210547Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_weather>
2020-02-09T18:17:58.2211182Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_weather, old_state=None, new_state=<state sensor.la_clusaz_weather=TrÃ¨s nuageux; attribution=Data provided by MÃ©tÃ©o-France, friendly_name=La-Clusaz Weather, icon=mdi:weather-partly-cloudy @ 2020-02-09T18:05:40.357537+00:00>>
2020-02-09T18:17:58.2211511Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_rain_chance
2020-02-09T18:17:58.2211820Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_rain_chance>
2020-02-09T18:17:58.2212501Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_rain_chance, old_state=None, new_state=<state sensor.la_clusaz_rain_chance=20; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Rain chance, icon=mdi:weather-rainy @ 2020-02-09T18:05:40.358920+00:00>>
2020-02-09T18:17:58.2212815Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_uv
2020-02-09T18:17:58.2213029Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_uv>
2020-02-09T18:17:58.2213701Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_uv, old_state=None, new_state=<state sensor.la_clusaz_uv=2; attribution=Data provided by MÃ©tÃ©o-France, friendly_name=La-Clusaz UV, icon=mdi:sunglasses @ 2020-02-09T18:05:40.359973+00:00>>
2020-02-09T18:17:58.2214001Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_freeze_chance
2020-02-09T18:17:58.2214217Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_freeze_chance>
2020-02-09T18:17:58.2214935Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_freeze_chance, old_state=None, new_state=<state sensor.la_clusaz_freeze_chance=0; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Freeze chance, icon=mdi:snowflake @ 2020-02-09T18:05:40.361123+00:00>>
2020-02-09T18:17:58.2215270Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_wind_speed
2020-02-09T18:17:58.2215495Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_wind_speed>
2020-02-09T18:17:58.2216135Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_wind_speed, old_state=None, new_state=<state sensor.la_clusaz_wind_speed=10; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=km/h, friendly_name=La-Clusaz Wind Speed, icon=mdi:weather-windy @ 2020-02-09T18:05:40.362469+00:00>>
2020-02-09T18:17:58.2216448Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_thunder_chance
2020-02-09T18:17:58.2216664Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_thunder_chance>
2020-02-09T18:17:58.2217322Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_thunder_chance, old_state=None, new_state=<state sensor.la_clusaz_thunder_chance=0; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Thunder chance, icon=mdi:weather-lightning @ 2020-02-09T18:05:40.363769+00:00>>
2020-02-09T18:17:58.2217859Z ERROR    homeassistant.components.meteo_france.sensor:sensor.py:166 No condition next_rain for location La-Clusaz
2020-02-09T18:17:58.2218089Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_next_rain
2020-02-09T18:17:58.2218308Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_next_rain>
2020-02-09T18:17:58.2218939Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_next_rain, old_state=None, new_state=<state sensor.la_clusaz_next_rain=unknown; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=min, friendly_name=La-Clusaz Next rain, icon=mdi:weather-rainy @ 2020-02-09T18:05:40.365052+00:00>>
2020-02-09T18:17:58.2219489Z DEBUG    homeassistant.components.meteo_france.sensor:sensor.py:156 weather alert watcher for La-Clusaz updated. Proxy have the status: checksum_cached_60s
2020-02-09T18:17:58.2219730Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_weather_alert
2020-02-09T18:17:58.2219941Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_weather_alert>
2020-02-09T18:17:58.2220764Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_weather_alert, old_state=None, new_state=<state sensor.la_clusaz_weather_alert=Jaune; Bulletin date=2020-02-09T15:05:00+00:00, Vent violent=Jaune, Pluie-innodation=Vert, Orages=Vert, Inondation=Vert, Neige-verglas=Vert, Canicule=Vert, Grand-froid=Vert, Avalanches=Jaune, Vagues-submersion=Vert, attribution=Data provided by MÃ©tÃ©o-France, friendly_name=La-Clusaz Weather Alert, icon=mdi:weather-cloudy-alert @ 2020-02-09T18:05:40.366510+00:00>>
2020-02-09T18:17:58.2221227Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_snow_chance
2020-02-09T18:17:58.2221452Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_snow_chance>
2020-02-09T18:17:58.2222164Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_snow_chance, old_state=None, new_state=<state sensor.la_clusaz_snow_chance=10; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=%, friendly_name=La-Clusaz Snow chance, icon=mdi:weather-snowy @ 2020-02-09T18:05:40.367746+00:00>>
2020-02-09T18:17:58.2222476Z INFO     homeassistant.helpers.entity_registry:entity_registry.py:220 Registered new sensor.meteo_france entity: sensor.la_clusaz_temperature
2020-02-09T18:17:58.2222699Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event entity_registry_updated[L]: action=create, entity_id=sensor.la_clusaz_temperature>
2020-02-09T18:17:58.2223358Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event state_changed[L]: entity_id=sensor.la_clusaz_temperature, old_state=None, new_state=<state sensor.la_clusaz_temperature=4; attribution=Data provided by MÃ©tÃ©o-France, unit_of_measurement=Â°C, friendly_name=La-Clusaz Temperature, icon=mdi:thermometer, device_class=temperature @ 2020-02-09T18:05:40.368917+00:00>>
2020-02-09T18:17:58.2223666Z DEBUG    homeassistant.core:core.py:570 Bus:Handling <Event homeassistant_close[L]>
```

## Additional information

--- a/tests/components/meteo_france/conftest.py
+++ b/tests/components/meteo_france/conftest.py
@@ -1,0 +1,12 @@
+"""Meteo-France generic test utils."""
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def patch_requests():
+    """Stub out services that makes requests."""
+    with patch("homeassistant.components.meteo_france.meteofranceClient"):
+        with patch("homeassistant.components.meteo_france.VigilanceMeteoFranceProxy"):
+            yield

--- a/tests/components/meteo_france/conftest.py
+++ b/tests/components/meteo_france/conftest.py
@@ -7,6 +7,10 @@ import pytest
 @pytest.fixture(autouse=True)
 def patch_requests():
     """Stub out services that makes requests."""
-    with patch("homeassistant.components.meteo_france.meteofranceClient"):
-        with patch("homeassistant.components.meteo_france.VigilanceMeteoFranceProxy"):
-            yield
+    patch_client = patch("homeassistant.components.meteo_france.meteofranceClient")
+    patch_weather_alert = patch(
+        "homeassistant.components.meteo_france.VigilanceMeteoFranceProxy"
+    )
+
+    with patch_client, patch_weather_alert:
+        yield

